nautobot/docs/additional-features/jobs.md

```diff
@@ -18,14 +18,14 @@ Jobs are a way for users to execute custom logic on demand from within the Nauto
 
 Jobs may be installed in one of three ways:
 
-- Manually installed as files in the [`JOBS_ROOT`](../../configuration/optional-settings/#jobs_root) path (which defaults to `$NAUTOBOT_ROOT/jobs/`).
-    - The `JOBS_ROOT` directory *must* contain a file named `__init__.py`. Do not delete this file.
-    - Each file created within this path is considered a separate module; there is no support for cross-file dependencies (such as a file acting as a common "library" module of functions shared between jobs) for files installed in this way.
-- Imported from an external [Git repository](../models/extras/gitrepository.md#jobs).
-    - The repository's `jobs/` directory *must* contain a file named `__init__.py`.
-    - Each Job file in the repository is considered a separate module; there is no support for cross-file dependencies (such as a file acting as a common "library" module of functions shared between jobs) for files installed in this way.
-- Packaged as part of a [plugin](../plugins/development.md#including-jobs).
-    - Jobs installed this way are part of the plugin module and can import code from elsewhere in the plugin or even have dependencies on other packages, if needed, via the standard Python packaging mechanisms.
+* Manually installed as files in the [`JOBS_ROOT`](../../configuration/optional-settings/#jobs_root) path (which defaults to `$NAUTOBOT_ROOT/jobs/`).
+  * The `JOBS_ROOT` directory *must* contain a file named `__init__.py`. Do not delete this file.
+  * Each file created within this path is considered a separate module; there is no support for cross-file dependencies (such as a file acting as a common "library" module of functions shared between jobs) for files installed in this way.
+* Imported from an external [Git repository](../models/extras/gitrepository.md#jobs).
+  * The repository's `jobs/` directory *must* contain a file named `__init__.py`.
+  * Each Job file in the repository is considered a separate module; there is no support for cross-file dependencies (such as a file acting as a common "library" module of functions shared between jobs) for files installed in this way.
+* Packaged as part of a [plugin](../plugins/development.md#including-jobs).
+  * Jobs installed this way are part of the plugin module and can import code from elsewhere in the plugin or even have dependencies on other packages, if needed, via the standard Python packaging mechanisms.
```

Comment:
![Screen Shot 2022-03-29 at 7 40 19 AM](https://user-images.githubusercontent.com/536467/160603677-9e0f5669-9237-4e52-a951-1dd17576dc72.png)

The left image is from the CI build (from this PR).  The right is from running `mkdocs serve` locally.
I'll get these reverted, but we'll want to look into these rendering differences sometime.

Reply:
https://markdown-guide.readthedocs.io/en/latest/basics.html#lists-nested
I think ReadtheDocs requires 4 spaces, however my local viewer will render indents with only 2 spaces, and MD007 in the markdown linter marks it as an error.  https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md007
I'll get all lists I've updated reverted back to 4 spaces, and update the linting settings accordingly.

Reply:
Good to know about the rendering differences! That's unfortunate. I don't know that we have a specific section in the docs about authoring of docs (if we did, that would be a nice place to add a note about this), but at the very least, if we can get linting to enforce 4-spaces consistently, that should help a lot.

Reply:
> Good to know about the rendering differences! That's unfortunate. I don't know that we have a specific section in the docs about authoring of docs (if we did, that would be a nice place to add a note about this), but at the very least, if we can get linting to enforce 4-spaces consistently, that should help a lot.

@glennmatthews I added in a `.markdownlint` config file that updates compatibility with our docs.  In there I specified 4 spaces instead of 2, and updated all nested lists on all other pages accordingly.  This issue should be fully resolved now.